helix-term/src/commands/typed.rs

```diff
@@ -672,7 +672,7 @@ pub fn write_all_impl(
             }
             if doc.path().is_none() {
                 if write_scratch {
-                    errors.push("cannot write a buffer without a filename\n");
+                    errors.push("cannot write a buffer without a filename");
```

Comment:
it doesn't seem like this change is needed anymore?

Reply:
oh I see I guess this is fine

Reply:
I just reverted it back